tests/utils/daemonset/daemonset.go

```diff
@@ -149,3 +151,20 @@ func RedefineWithVolumeMount(daemonSet *v1.DaemonSet) *v1.DaemonSet {
 
 	return daemonSet
 }
+
+func RedefineWithResources(daemonSet *v1.DaemonSet, limit string, req string) {
```

Comment:
There are other resources (cpu & hugepages) that can be set, but you're only setting the cpu. I suggest the name "RedefineWithCpuResources" instead.

Reply:
Fixed, thanks